doc/mssqlserver/doc.md

```diff
@@ -0,0 +1,145 @@
+Galleon Layers
+=========
+
+* `mssqlserver-datasource`: Provision the `MSSQLServerDS` non xa datasource. Depends on `mssqlserver-jre8-driver` layer.
+* `mssqlserver-jre8-driver`: Provision the `mssqlserver-jre8` driver.
+
+Configuration
+========
+
+The following set of environment variables and corresponding Java system properties can be used to configure the datasource.
+
+Required configuration
+==============
+
+* `MSSQLSERVER_DATABASE`
+
+  * Description: Defines the database name to be used in the datasource’s `connection-url` property.
+  * Default Value: `NONE`
```

Comment:
I'm nitpicking but does that mean that there is no default value or that the default value is the `NONE` string :)

If there is no default value, it might be more explicit with `* No default value`

Reply:
Ok.